src/libraries/Native/Unix/System.Native/config.h.in

```diff
@@ -0,0 +1,7 @@
+#ifndef SYSTEMNATIVE_CONFIG_H_INCLUDED
```

Comment:
The rest of System.Native is using the configure from `src\libraries\Native\Unix`. I think we should continue with that plan and just add these two defined to `src\libraries\Native\Unix`. Or is there a reason for this being in its own configure script?

Reply:
No, I had just missed that one already existed a directory up, will update.

Reply:
Should be fixed now.